src/cloc/workflow.yaml

```diff
@@ -13,7 +13,7 @@ volumes:
     readOnly: true
 activities:
   cloc:
-    image: aldanial/cloc
+    image: aldanial/cloc:1.94
```

Comment:
I changed the version to the stable one. For some reason, the latest version doesn't work. Could you please double-check the latest version and revert these changes if everything works?